ansible/roles/install_git/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Install Git
+  become: true
+
+  block:
+    - name: Create directory /home/authz/tmp
+      file:
+        path: /home/authz/tmp
+        state: directory
+
+    - name: Update apt-cache
+      apt:
+        update_cache: yes
+        cache_valid_time: 3600
+
+    - name: Install required packages for compiling Git
+      apt:
+        name:
+          - libz-dev
+          - libssl-dev
+          - libcurl4-gnutls-dev
+          - libexpat1-dev
+          - gcc
+          - gettext
+          - cmake
+        state: present
+
+    - name: Download Git 2.40.1 
+      unarchive:
+        src: https://github.com/git/git/archive/v2.40.1.tar.gz
+        dest: /home/authz/tmp
+        remote_src: true
+
+    - name: Install Git 2.40.1 
+      shell: |
+        cd /home/authz/tmp/git-2.40.1
```

Comment:
we need a way to install git only once. Now its installed every time (although `make` checks that all needed targets are compiled already)

Reply:
Before downloading the git, I added a check for the git executable file. If the file exists, the download and installation of the git will be skipped. If you meant another implementation or what is my problem, then please explain.

Reply:
Hmm i see. So yeah the code to skip the task is there let me double check if its working or not

Reply:
yeah seems its getting skipped. LGTM